### PR TITLE
chore: first pass at docs for the delivery firehose

### PR DESCRIPTION
**Pull request by travis**
I think these need more work, but want to see how this lands.

### CODE REVIEW

File: docs/ucan-invocation-stream.md [hunk 1]
```diff
@@ -53,3 +53,33 @@ Consumers might need other infrastructure resources to track state based on the
...
+"op" (ie, the UCAN's ability - `store/add`, `upload/remove`, etc) and "day"
+(a `%Y-%m-%d` formatted string). We designed partitioning this way to make it easy to 
+find logs for a particular operation on a particular day, which is used extensively
+by AWS Athena to make UCAN log queries efficient.
+
+For example, receipts of the `store/add` operation from January 1, 2024 are stored in:
+
+`/logs/receipt/store/add/2024-01-01`
+
+#### Glue
```
Comment (gobengo) [inline docs/ucan-invocation-stream.md:70]: could this link to where glue-related code is in w3infra? (if any). glue is new to me

Comment (travis) [reply docs/ucan-invocation-stream.md:70]: ah great idea, done!

File: docs/ucan-invocation-stream.md [hunk 2]
```diff
@@ -53,3 +53,33 @@ Consumers might need other infrastructure resources to track state based on the
 The `admin-metrics` table has a partition key `name` with the metric name we keep track. With this, we can easily update and query each of the `admin` metrics we care about.
 
 In the context of `space-metrics` table, a partition key with `space` is used together with a sort key `name` with the metric name. This way, we are able to track and query each metric for a given space.
+
+### Delivery Firehose
+
+At the end of the invocation stream we archive UCAN invocations and receipts as JSON
+logs stored in S3. We [partition log storage](https://github.com/web3-storage/w3infra/blob/9def8df1ac3e0dda6e7aad710b1ec534af50af0a/stacks/firehose-stack.js#L163) by "type" (ie, `workflow` or `receipt`),
```
Comment (vasco-santos) [inline docs/ucan-invocation-stream.md:60]:
```suggestion
Each UCAN Invocation/receipt that goes through the stream is stored 
in S3. We [partition log storage](https://github.com/web3-storage/w3infra/blob/9def8df1ac3e0dda6e7aad710b1ec534af50af0a/stacks/firehose-stack.js#L163) by "type" (ie, `workflow` or `receipt`),
```

Comment (vasco-santos) [reply docs/ucan-invocation-stream.md:60]: nit: Think this reads nicely

File: docs/ucan-invocation-stream.md [hunk 3]
```diff
@@ -53,3 +53,33 @@ Consumers might need other infrastructure resources to track state based on the
...
+"op" (ie, the UCAN's ability - `store/add`, `upload/remove`, etc) and "day"
+(a `%Y-%m-%d` formatted string). We designed partitioning this way to make it easy to
```
Comment (vasco-santos) [inline docs/ucan-invocation-stream.md:62]:
```suggestion
(a `%Y-%m-%d` formatted string). We designed partitioning this way to make it easy and efficient to 
```

File: docs/ucan-invocation-stream.md [hunk 4]
```diff
@@ -53,3 +53,33 @@ Consumers might need other infrastructure resources to track state based on the
...
 
 In the context of `space-metrics` table, a partition key with `space` is used together with a sort key `name` with the metric name. This way, we are able to track and query each metric for a given space.
+
+### Delivery Firehose
+
+At the end of the invocation stream we archive UCAN invocations and receipts as JSON
+logs stored in S3. We [partition log storage](https://github.com/web3-storage/w3infra/blob/9def8df1ac3e0dda6e7aad710b1ec534af50af0a/stacks/firehose-stack.js#L163) by "type" (ie, `workflow` or `receipt`),
+"op" (ie, the UCAN's ability - `store/add`, `upload/remove`, etc) and "day"
+(a `%Y-%m-%d` formatted string). We designed partitioning this way to make it easy to 
+find logs for a particular operation on a particular day, which is used extensively
```
Comment (vasco-santos) [inline docs/ucan-invocation-stream.md:63]:
```suggestion
find logs for a particular operation on a particular date, which is used extensively
```

File: docs/ucan-invocation-stream.md [hunk 5]
```diff
@@ -53,3 +53,33 @@ Consumers might need other infrastructure resources to track state based on the
...
+by AWS Athena to make UCAN log queries efficient.
+
+For example, receipts of the `store/add` operation from January 1, 2024 are stored in:
+
+`/logs/receipt/store/add/2024-01-01`
+
+#### Glue
+
+We [use AWS Glue](https://github.com/web3-storage/w3infra/blob/9def8df1ac3e0dda6e7aad710b1ec534af50af0a/stacks/firehose-stack.js#L171) to defined "tables" that take advantage of the partitioning structure
```
Comment (vasco-santos) [inline docs/ucan-invocation-stream.md:72]:
```suggestion
We [use AWS Glue](https://github.com/web3-storage/w3infra/blob/9def8df1ac3e0dda6e7aad710b1ec534af50af0a/stacks/firehose-stack.js#L171) to define "tables" that take advantage of the partitioning structure
```

**Review by vasco-santos — APPROVED** [review]
Thanks ❤️

**Pull request merged**